### PR TITLE
Make tests work with Karma and Chrome (Closes #60)

### LINKED ISSUE #60: Write tests for the calendar

### PR DESCRIPTION
**Pull request by julienw**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -97,6 +97,6 @@
     "sinon-chai": "^2.8.0",
     "stylelint": "^6.5.1",
     "stylelint-config-standard": "^8.0.0",
-    "uglify-js": "github:mishoo/UglifyJS2#harmony"
+    "uglify-js": "github:mishoo/UglifyJS2#766fafda8baefe6d150df27dfad7e99857c08557"
```
Comment (julienw) [inline package.json:101]: waiting for https://github.com/mishoo/UglifyJS2/pull/1229

File: karma.conf.js [hunk 2]
```diff
@@ -19,12 +19,19 @@ module.exports = function(config) {
...
+      { pattern: 'node_modules/twitter_cldr/min/*.js', included: false },
       'tests/unit/test-main.js',
     ],
 
     // Test results reporter to use.
     reporters: ['mocha'],
 
+    mochaReporter: {
+      showDiff: true,
+    },
```
Comment (julienw) [inline karma.conf.js:33]: we can have nice things

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  apt:
+    sources:
+      google-chrome
+    packages:
+      google-chrome-stable
+before_install:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - sleep 3 # give xvfb some time to start
+  - export CHROME_BIN=google-chrome
```
Comment (julienw) [inline .travis.yml:14]: otherwise an older chromium is used

File: package.json (hunk 1 above)
Comment (samgiles) [inline package.json:101]: nit: Can we make a GitHub issue and mention the related issue?

**Comment by samgiles** [comment]
Looks good to me! And it passed on Travis! Woohoo, nice work!  I'll merge.

[Sorry for the delay :| ]

File: package.json (hunk 1 above)
Comment (julienw) [inline package.json:101]: filed https://github.com/fxbox/calendar/issues/68

**Pull request merged**